maa.jwt.verifier/include/base64.hpp

```diff
@@ -0,0 +1,26 @@
+/***************************************************************************************
```

Comment:
base64 encryption code was based off of https://github.com/mvorbrodt/blog/blob/master/src/base64.hpp